### PR TITLE
Clean up error naming and list issues 

### PR DESCRIPTION
**Pull request by pzhanggit**
- [x] Fix incorrect "MAE", "MSE" naming to "RMSE" 
- [x] Remove error list may not be needed, e.g., error of sum, anymore
- [x] Remove list append

### CODE REVIEW

File: hydragnn/train/train_validate_test.py [hunk 1]
```diff
@@ -70,9 +70,9 @@ def train_validate_test(
     visualizer.num_nodes_plot()
 
     if plot_init_solution:  # visualizing of initial conditions
-        test_rmse = test(test_loader, model, verbosity)
-        true_values = test_rmse[3]
-        predicted_values = test_rmse[4]
+        test_res = test(test_loader, model, verbosity)
```
Comment (streeve) [inline hydragnn/train/train_validate_test.py:73]: Why 'res'?

**Comment by streeve** [comment]
Replacing list appending seems like a big job - separate PR?

**Comment by pzhanggit** [comment]
> Replacing list appending seems like a big job - separate PR?

I am just going to replace the appending in error lists in this PR.

File: hydragnn/train/train_validate_test.py (hunk 1 above)
Comment (streeve) [reply hydragnn/train/train_validate_test.py:73]: Got it. `test_results`, please :)

Comment (pzhanggit) [reply hydragnn/train/train_validate_test.py]: Done.

**Comment by streeve** [comment]
So to be clear, we just don't care about the node sum error anymore?

**Comment by pzhanggit** [comment]
> So to be clear, we just don't care about the node sum error anymore?

Yes, especially given the good results @allaffa got. At one point, we can only get good predictions for sum of node values and had the discussion about loss of node sum. But I do not think we need it any more.

File: hydragnn/train/train_validate_test.py [hunk 2]
```diff
@@ -70,9 +66,9 @@ def train_validate_test(
     visualizer.num_nodes_plot()
 
     if plot_init_solution:  # visualizing of initial conditions
-        test_rmse = test(test_loader, model, verbosity)
-        true_values = test_rmse[3]
-        predicted_values = test_rmse[4]
+        test_results = test(test_loader, model, verbosity)
```
Comment (streeve) [inline hydragnn/train/train_validate_test.py:69]: Maybe we just return all the values separately here too? I think will be easier to read

Comment (pzhanggit) [reply hydragnn/train/train_validate_test.py:69]: Done.

**Review by streeve — APPROVED** [review]
Looks good to me

**Comment by streeve** [comment]
@allaffa okay to merge?

**Review by allaffa — APPROVED** [review]

**Pull request merged**